From 7065669143c846ddb523ccb7cd214458e246ff3c Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Thu, 3 Oct 2024 07:35:22 -0400
Subject: [PATCH] scripts: Ignore filesystem.transfiletriggerin

This is all about migrations and hacks, which we don't need
since every filesystem tree is constructed fresh.

xref https://gitlab.com/fedora/bootc/tracker/-/issues/29

Signed-off-by: Colin Walters <walters@verbum.org>
---
 rust/src/scripts.rs | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/rust/src/scripts.rs b/rust/src/scripts.rs
index 34c34696..e5ac83fe 100644
--- a/rust/src/scripts.rs
+++ b/rust/src/scripts.rs
@@ -77,6 +77,9 @@ static IGNORED_PKG_SCRIPTS: phf::Set<&'static str> = phf_set! {
     "systemd.transfiletriggerin",
     // https://bugzilla.redhat.com/show_bug.cgi?id=1473402
     "man-db.transfiletriggerin",
+    // See https://gitlab.com/fedora/bootc/tracker/-/issues/29 - we don't need
+    // any of this.
+    "filesystem.transfiletriggerin",
     // https://src.fedoraproject.org/rpms/nfs-utils/pull-request/1
     "nfs-utils.post",
     // There is some totally insane stuff going on here in RHEL7
-- 
2.44.0

